src/main/api/studio-api.yaml

```diff
@@ -6531,33 +6242,110 @@ paths:
         '500':
           $ref: '#/components/responses/InternalServerError'
 
-  /api/2/publish/all:
+  /api/2/publish/{siteId}/dependencies:
```

Comment:
Change to `calculate_package`

Reply:
Let's consider /publish/{siteId}/calculate